go/vt/vtctl/vtctl.go

```diff
@@ -2351,8 +2351,8 @@ func commandVerticalSplitClone(ctx context.Context, wr *wrangler.Wrangler, subFl
 }
 
 func commandVDiff(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
-	sourceCell := subFlags.String("source_cell", "", "The source cell to compare from")
-	targetCell := subFlags.String("target_cell", "", "The target cell to compare with")
+	sourceCell := subFlags.String("source_cell", "", "The source cell to compare from; default is all cells")
+	targetCell := subFlags.String("target_cell", "", "The target cell to compare with; default is all cells")
```

Comment:
Isn't all cells misleading? we would choose one out of all available.

Reply:
I wasn't trying to comment on the effect of the default setting;  just that the default is all cell cells. Will adjust.  The docs have the same issue, BTW.